airflow/api_connexion/openapi/v1.yaml

```
@@ -1091,6 +1091,29 @@ paths:
         "404":
           $ref: "#/components/responses/NotFound"
 
+  /dag/parse/{file_token}:
```

```suggestion
  /parseDagFie/{file_token}:
```
Or something similar. Note the dag source endpoint is `/dagSources`.

---

Updated the URL for API